fpsim/locations/ethiopia.py

```diff
@@ -384,6 +425,19 @@ def lactational_amenorrhea():
     return lactational_amenorrhea
 
 
+def lactational_amenorrhea_region():
+    '''
+    Returns an array of the percent of breastfeeding women by month postpartum 0-11 months who meet criteria for LAM, , stratified by region
+    Uses 2016 Ethiopia DHS individual recode (v025) for region and V024 for urban to produce subnational estimates
+    '''
+    lactational_amenorrhea_region = pd.read_csv(thisdir / 'ethiopia' / 'subnational' / 'lam_region.csv')
```

Comment:
since you're reading in the regional lam file and returning a separate object (a dictionary of regional lam values), it's best practice to name these two variables differently

Reply:
Thanks for pointing this out! I have updated the variable names accordingly.